### PR TITLE
editPlayer, Notes, and notesView update

### PR DESCRIPTION
**Pull request by itsJarrett**
These commits do the following: 

- Fix editPlayer.php </form> tag out of bounds.
- Update Player ID to show in first column (correlates with Players view)
- Changes "Warning Type" to "Note Type"
- Added "Commendation" Note Type
- Update verifyCheck.php to withstand the "Commendation" Note Type update.

### CODE REVIEW

**Comment by Jason2605** [comment]
I haven't had a chance to test this, can you please confirm it works, if not ill have a chance to test tomorrow maybe.

**Comment by itsJarrett** [comment]
Yes it works, I have a local copy running for debugging.

File: notesView.php [hunk 1]
```diff
@@ -91,14 +91,22 @@
...
-        echo '<tr class = "warning">';
-    } elseif ($row['warning'] == 3) {
-        echo '<tr class = "danger">';
-    } elseif ($row['warning'] == 4) {
-        echo '<tr class = "success">';
-    } else {
-        echo '<tr>';
+    switch ($row['warning']) {
+        case 1:
+            echo '<tr class = "warning">';
```
Comment (Jason2605) [inline notesView.php]: case 1 should be the default, as in no colour, not red

File: notesView.php [hunk 2]
```diff
@@ -78,29 +78,36 @@
...
 while ($row = mysqli_fetch_array($search_result, MYSQLI_ASSOC)) {
     echo '<form action=notesView.php method=post>';
-    if ($row['warning'] == 2) {
-        echo '<tr class = "warning">';
-    } elseif ($row['warning'] == 3) {
-        echo '<tr class = "danger">';
-    } else {
-        echo '<tr>';
+    switch ($row['warning']) {
+        case 1:
```
Comment (Jason2605) [inline notesView.php]: I would make case 1 the default case

**Comment by Jason2605** [comment]
Nice work.

**Pull request merged**